model/channel_member.go

```diff
@@ -156,6 +156,27 @@ func IsIgnoreChannelMentionsValid(ignoreChannelMentions string) bool {
 	return ignoreChannelMentions == IgnoreChannelMentionsOn || ignoreChannelMentions == IgnoreChannelMentionsOff || ignoreChannelMentions == IgnoreChannelMentionsDefault
 }
 
+func ShouldChannelMemberNotifyCRT(notifyProps StringMap, isMentioned bool) (bool, bool) {
```

Comment:
Let's make this a private function inside app/notification.go. There's no need to expose this in model.

Also, let's use named returns `notifyDesktop, notifyPush bool` for readability.